packages/ui/components/createButton/CreateButton.tsx

```diff
@@ -0,0 +1,128 @@
+import { useRouter } from "next/router";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import {
+  Avatar,
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuTrigger,
+} from "@calcom/ui";
+import { FiPlus } from "@calcom/ui/components/icon";
+
+export interface Parent {
```

Comment:
I think we may call it `Option` only and accept just `image` and 'label'. This keeps the component simpler and less brittle and more reusable.
User of this component can then decide if `image` is to be computed from` profile.image` or `profile.slug`. If label should be `profile.name` or `profile.slug`.

Also, `teamId` is like the option value, whatever option is selected, corresponding option might be made available in `createFunction` and then createFunction callback can do whatever it needs(e.g. it needs to decide if it's team or not and if it's a team which teamId)

```javascript
[
{
value: -1, // User
label: '',
image: ''
},
{
value: 1, //Team with ID 1
label: '',
image: ''
},]
```